devices/org.osbuild.loopback

```diff
@@ -106,7 +108,8 @@ class LoopbackService(devices.DeviceService):
         dir_fd = -1
         try:
             dir_fd = os.open(devpath, os.O_CLOEXEC | os.O_PATH)
-            self.lo.mknod(dir_fd)
+            with ctx.suppress_oserror(errno.EEXIST):
+                self.lo.mknod(dir_fd)
```

Comment:
I guess you don't use `ensure_device_node()` here, because you want to call into `self.lo`? Sounds fine.

Reply:
Yeah, although I was pondering replacing it as well.

Reply:
I think either way would be fine!